src/core_landice/mode_forward/mpas_li_time_integration_fe.F

```diff
@@ -140,12 +146,6 @@ subroutine li_time_integrator_forwardeuler(domain, err)
           call mpas_timer_stop("damage")
       endif
 
-! === Compute new state for prognostic variables ==================================
-      call mpas_timer_start("advect thickness and tracers")
-      call advection_solver(domain, err_tmp)
-      err = ior(err, err_tmp)
-      call mpas_timer_stop("advect thickness and tracers")
-
 ! === preserve high damage ===========
       if (config_preserve_high_damage) then
```

Comment:
Now that both damage routines are called right after each other, they should be combined into a single routine.

Reply:
I now  rename li_preserve_high_damage to li_finalize_damage_after_advection to include the GL damage setting, damage preserve (healing) and rheology-damage coupling.